Alters "Get Started" section (#1065)
iotaledger/iota-wiki: docs/get-started/introduction/iota/iota-token.md

---
description: The IOTA token is designed to facilitate value transfer between humans and machines, with fair distribution and a focus on sustainability. It enables feeless transactions and supports a wide range of use cases.
image: /img/logo/Chrysalis_logo_dark.png
keywords:
  - IOTA token
  - value transfer
  - fair distribution
  - non-profit foundation
  - feeless transactions
  - buying IOTA
  - IOTA cryptocurrency
  - major exchanges
---

# IOTA Token

> The IOTA token acts as the medium to do value transfers within the IOTA mainnet. However, unlike other cryptos

Reviewer: ```suggestion
The IOTA token is the medium to transfer value within the IOTA mainnet. However, unlike other cryptos
```

there are no fees to be paid with the base token, meaning that IOTA tokens are transferred 1 to 1 to the receiver,
without miners or other intermediaries taking a cut.

## Fair Distribution & No Inflation

> Unlike most other cryptocurrencies, all 2,779,530,283,277,76 IOTA tokens have been fairly distributed up on its

Reviewer: ```suggestion
Unlike most other cryptocurrencies, all 2,779,530,283,277,76 IOTA tokens have been fairly distributed upon its
```

initial ICO without any early VC funding or tokens allocated to founders. Furthermore, there are no vesting schedules

> or token lockups, meaning that the entire supply is circulating.

Reviewer: ```suggestion
or token lockups, meaning the entire supply is circulating.
```

> Also unlike other protocols, IOTA has no inflation baked into its protocol, meaning that token holders are not getting

Reviewer: ```suggestion
Unlike other networks, IOTA has no inflation baked into its protocol, meaning that token holders are not getting
```
or 
```suggestion
IOTA has no inflation baked into its protocol, meaning that token holders are not getting
```
mainly to avoid the protocol's protocol
